cmd/commands/subvolume.go

```diff
@@ -30,12 +30,14 @@ var SubvolumeCmd = &cobra.Command{
 }
 
 var listCmd = &cobra.Command{
-	Use:   "ls",
-	Short: "Print the list of subvolumes.",
+	Use:     "ls",
+	Short:   "Print the list of subvolumes.",
+	Example: "kubectl rook-ceph subvolume ls --svg svgname --stale",
```

Comment:
```suggestion
	Example: "kubectl rook-ceph subvolume ls",
```
Let's have simple example for the command?